### PR TITLE
User authorization+template

### PR DESCRIPTION
**Pull request by Mo1far**
added function login/logout 
some fix with .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,7 @@ wheels/
 *.egg-info/
 .installed.cfg
 *.egg
+.idea/
```
Comment (kintawer) [inline .gitignore]: add it under `# ideas`

File: app/__init__.py [hunk 2]
```diff
@@ -16,7 +17,7 @@
 # database
 db = SQLAlchemy(app)
 migrate = Migrate(app, db)
-
```
Comment (xzwache) [inline app/__init__.py]: add comment "authorization"

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kintawer) [inline .idea/vcs.xml]: it should be in .gitignore

File: app/forms.py [hunk 4]
```diff
@@ -1 +1,10 @@
 # Here will be Flask Web Forms
+from flask_wtf import  FlaskForm
+from wtforms import  StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import DataRequired
+
+class LoginForm(FlaskForm):
+    username = StringField("Username", validators=[DataRequired()])
+    password = PasswordField("Password", validators=[DataRequired()])
+    remember_me = BooleanField("Remember?")
+    submit = SubmitField("Sign in")
```
Comment (kintawer) [inline app/forms.py]: need empty line in the end of file (PEP8)

**Review by xzwache — APPROVED** [review]

File: app/forms.py [hunk 5]
```diff
@@ -1 +1,10 @@
 # Here will be Flask Web Forms
+from flask_wtf import  FlaskForm
+from wtforms import  StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import DataRequired
+
```
Comment (kintawer) [inline app/forms.py:4]: double blank lines (PEP8)

File: app/routes.py [hunk 6]
```diff
@@ -1,9 +1,45 @@
...
+
+
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+    if current_user.is_authenticated:
+        return redirect(url_for('index'))
+    form = LoginForm()
+    if form.validate_on_submit():
+        user = User.query.filter_by(username=form.username.data).first()
+        print(user)
```
Comment (kintawer) [inline app/routes.py]: print?

File: app/forms.py [hunk 7]
```diff
@@ -1 +1,10 @@
...
+class LoginForm(FlaskForm):
+    username = StringField("Username", validators=[DataRequired()])
```
Comment (CHuKeR) [inline app/forms.py]: Are we from England? If I remember we make ru-app :)

File: app/routes.py (hunk 6 above)
Comment (CHuKeR) [inline app/routes.py]: What's it? Delete print function.

File: app/templates/base.html [hunk 8]
```diff
@@ -1,4 +1,46 @@
-<!-- HERE WILL BE BOOTSTRAP TEMPLATE-->
+<!DOCTYPE html>
```
Comment (CHuKeR) [inline app/templates/base.html]: Why do you commit it here? It should be other commit like "Add base.html".

**Review by CHuKeR — APPROVED** [review]

**Pull request merged**